graphql-test-utils/index.js

```diff
@@ -1,14 +1,7 @@
-import * as mocksForMc from './graphql-mocks/mc';
-import * as mocksForProxy from './graphql-mocks/proxy';
-import * as mocksForSettings from './graphql-mocks/settings';
-
-export { default as applyMocksForExternalNetworkRequests } from './apply-mocks-for-external-network-requests';
-export { default as createGraphqlMockServer } from './create-graphql-mock-server';
 export { default as UserMock } from './graphql-models/user';
 export { default as ProjectMock } from './graphql-models/project';
 export { default as ApplicationAppbarMenuMock } from './graphql-models/application-appbar-menu';
 export { default as ApplicationNavbarMenuMock } from './graphql-models/application-navbar-menu';
 export { default as ApplicationNavbarSubmenuMock } from './graphql-models/application-navbar-submenu';
 export { default as CustomApplicationMock } from './graphql-models/custom-application';
 export { default as ProjectExtensionMock } from './graphql-models/project-extension';
```

Comment:
FYI: As a follow up, we will be start open sourcing our internal test data, starting with the core library. With that, we can then build up the models here in appkit.

Reply:
Sure. That'd be cool.